content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -85,26 +86,27 @@ rules:
 - level: Metadata
 ```
 
-The audit profile used by GCE should be used as reference by admins constructing their own audit profiles. You can check the
+If you're crafting your own audit profile, you can use the audit profile for Google Container-Optimized OS as a starting point. You can check the
 [configure-helper.sh](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh)
-script, which generates the audit policy file. You can see most of the audit policy file by looking directly at the script.
+script, which generates an audit policy file. You can see most of the audit policy file by looking directly at the script.
 
 ## Audit backends
 
 Audit backends persist audit events to an external storage.
-[Kube-apiserver][kube-apiserver] out of the box provides two backends:
+Out of the box, the kube-apiserver provides two backends:
```

Comment:
Not a blocker, but worth noting:

I'd like to update the style guide specifically to avoid colloquialisms. When was the last time anyone opened a physical box to retrieve software? 🤷